homeassistant/components/tuya/select.py

```diff
@@ -342,6 +342,28 @@
             translation_key="countdown",
         ),
     ),
+    # Electric Blanket
+    # https://developer.tuya.com/en/docs/iot/categorydr?id=Kaiuz22dyc66p
+    "dr": (
+        SelectEntityDescription(
+            key=DPCode.LEVEL,
+            name="Level",
+            icon="mdi:thermometer-lines",
+            translation_key="blanket_level",
+        ),
+        SelectEntityDescription(
+            key=DPCode.LEVEL_1,
+            name="Side A Level",
+            icon="mdi:thermometer-lines",
+            translation_key="blanket_level",
+        ),
+        SelectEntityDescription(
+            key=DPCode.LEVEL_2,
+            name="Side B Level",
+            icon="mdi:thermometer-lines",
+            translation_key="blanket_level",
+        ),
```

Comment:
These DPs aren't in the linked documentation?

Reply:
My apologies for not making a stronger connection between the DPCodes and the documentation.  I was also frustrated by the complete mismatch of the documentation to what the diagnostics showed as the used codes.  I've attached the diagnostics report from this device from my Home Assistant.  

What would the correct way to support this device be?  Create local constants instead of adding to DPCodes?


[tuya-d49601845676c22b15d59c2008f4b0ef-Sunbeam Bedding-b547a8326b91fe6f46c2167bc036198e.json (1).txt](https://github.com/home-assistant/core/files/10495472/tuya-d49601845676c22b15d59c2008f4b0ef-Sunbeam.Bedding-b547a8326b91fe6f46c2167bc036198e.json.1.txt)